engineering/conventions-go.md

```diff
@@ -0,0 +1,62 @@
+
+# Go Development Conventions
+
+## Introduction
+
+This guide documents development conventions for Go at source{d}. Check [general conventions](conventions.md) for language-independent conventions that are also applicable for Go projects.
+
+## Supported Go Versions
+
+* Support latest two stable major versions of Go (e.g. 1.9.x and 1.10.x). Both versions should be included in Travis CI.
+  * Some of our projects may support more versions, specially when there is wide-adoption outside source{d} (e.g. [go-git](https://github.com/src-d/go-git) currently supports three).
+  * We try to support new Go versions as fast as possible. Versions to test in Travis CI include `tip` to help us identifying problems with next version ahead of time.
```

Comment:
Also, include wording about `allowed_failure`?

Reply:
Added it.

Anyway, I think that we should have examples / templates in [src-d/ci](https://github.com/src-d/ci) or in a project template repository. These would clarify Travis / Drone config to the detail.